### PR TITLE
Updating doc with UI text changes

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Hi, @eharris369, I saw your changes in this PR https://github.com/eclipse/codewind-eclipse/pull/402 and tried to make corresponding changes in the docs.

### CODE REVIEW

File: docs/_documentations/mdt-eclipse-managingprojects.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can work with your Codewind projects from the **Codewind Explorer** view in
 - Start typing **Codewind** in the filter field or locate and expand the **Codewind** entry in the list.
 - Select **Codewind Explorer** and click **Open**.
 
-To create a new project or import an existing one, use the context menu on the **Local Projects** item in the **Codewind Explorer** view.  Once you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging.
+To create a new project or import an existing one, use the context menu on the **Local** item in the **Codewind Explorer** view. After you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging.
```
Comment (eharris369) [inline docs/_documentations/mdt-eclipse-managingprojects.md]: I may have missed this in the readme but the projects are now imported automatically for the user so delete: "After you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging."

**Comment by eharris369** [comment]
Just the one comment.  Thank you for doing these changes!

File: docs/_documentations/mdt-eclipse-managingprojects.md (hunk 1 above)
Comment (eharris369) [reply docs/_documentations/mdt-eclipse-managingprojects.md]: I checked the readme and it is ok.

**Comment by sishida** [comment]
@eharris369 Nice catch! I have removed the obsolete lines. 😄

**Pull request merged**